drivers/bridge/setup_device.go

```diff
@@ -31,7 +32,29 @@ func setupDevice(config *NetworkConfiguration, i *bridgeInterface) error {
 	}
 
 	// Call out to netlink to create the device.
-	return netlink.LinkAdd(i.Link)
+	if err = netlink.LinkAdd(i.Link); err != nil {
+		return types.InternalErrorf("Failed to program bridge link: %s", err.Error())
+	}
+
+	if config.FixedCIDRv6 != nil {
```

Comment:
@mrjana there is a reason for this. Please check the old code and there is a TODO for the other case and it was intentionally not addressed.